sdks/python/scripts/run_pytest.sh

```diff
@@ -42,10 +42,10 @@ echo "posargs: $posargs"
 
 # Run with pytest-xdist and without.
 pytest -o junit_suite_name=${envname} \
-  --junitxml=pytest_${envname}.xml -m 'not no_xdist' -n 6 ${pytest_args} --pyargs ${posargs}
+  --junitxml=pytest_${envname}.xml -m 'not no_xdist' -n 6 --import-mode=importlib ${pytest_args} --pyargs ${posargs}
```

Comment:
Newer projects needs to use importlib. Also using importlib, we are able to use installed version of Apache Beam for testing in tox environments instead of the local source(without this flag installed version gets shadowed by the local source)

https://docs.pytest.org/en/7.1.x/explanation/goodpractices.html#which-import-mode

Reply:
do we need to do anything for test files with the same name (we have quite a bit of `util_test.py`)